cms/static/js/models/course_update.js

```diff
@@ -3,7 +3,9 @@ define(["backbone", "jquery", "jquery.ui"], function(Backbone, $) {
     var CourseUpdate = Backbone.Model.extend({
         defaults: {
             "date" : $.datepicker.formatDate('MM d, yy', new Date()),
-            "content" : ""
+            "content" : "",
+            "push_notification_enabled": false,
```

Comment:
Not knowing anything about backbone, are these the defaults? Presumably these are getting populated with the actual values somewhere?